spring-data-rest-webmvc/src/main/java/org/springframework/data/rest/webmvc/RepositoryRestHandlerMapping.java

```diff
@@ -266,14 +266,14 @@ public String resolveStringValue(String value) {
 	static class RepositoryCorsConfigurationAccessor {
 
 		private final @NonNull ResourceMappings mappings;
-		private final @NonNull Repositories repositories;
+		private final Repositories repositories;
 		private final @NonNull StringValueResolver embeddedValueResolver;
```

Comment:
Let's change the order (`StringValueResolver` before `Repositories`) so that we also have the constructor taking all non-null properties first.